trackintel/preprocessing/staypoints.py

```diff
@@ -243,7 +240,7 @@ def merge_staypoints(staypoints, triplegs, max_time_gap="10min", agg={}):
 
     Returns
```

Comment:
There shall be a `Staypoints.validate(staypoints)` line for this function?

Reply:
Yes, I think this is a good place. I added it.